spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/CassandraSQLContext.scala

```diff
@@ -36,199 +31,48 @@ import collection.mutable
   *
   * }}} */
 class CassandraSQLContext(sc: SparkContext) extends SQLContext(sc) {
```

Comment:
CSQContext(This name can be any name) is an implicit class holder to add additional methods to SQLContext(a more general use), the class is not used to be called directly . CassandraSqlContext is used for CassandraCatalog(only narrow used), it should be called directly. They are different.